File: examples/99.cs
using System;
class B {
  static void Main() {
    for (int n=99;n>0;n--) {
      Console.WriteLine("{0} bottle{1} of beer on the wall, {0} bottle{1} of beer.", n, n==1?"":"s");
      if (n-1>0)
        Console.WriteLine("Take one down and pass it around, {0} bottle{1} of beer on the wall.\n", n-1, n-1==1?"":"s");
      else
        Console.WriteLine("Take one down and pass it around, no more bottles of beer on the wall.\n");
    }
    Console.WriteLine("No more bottles of beer on the wall, no more bottles of beer.");
    Console.WriteLine("Go to the store and buy some more, 99 bottles of beer on the wall.");
  }
}

File: examples/99.rs
fn main() {
    for n in (1..=99).rev() {
        println!("{} bottle{} of beer on the wall, {} bottle{} of beer.", n, if n==1 {""} else {"s"}, n, if n==1 {""} else {"s"});
        if n-1 > 0 {
            println!("Take one down and pass it around, {} bottle{} of beer on the wall.\n", n-1, if n-1==1 {""} else {"s"});
        } else {
            println!("Take one down and pass it around, no more bottles of beer on the wall.\n");
        }
    }
    println!("No more bottles of beer on the wall, no more bottles of beer.");
    println!("Go to the store and buy some more, 99 bottles of beer on the wall.");
}
